examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+
+## Installation
+
+### Prerequisities
+
+* A functional Kubernetes cluster (autopilot also should be working)
+* kubectl
+* Docker
+* GNU Make
+* git
+* go
+* helm
+
+### Installing CLoudRetro
+
+Lets create a new namespace for CloudRetro;
+
+```console
+kubectl create namespace cloudretro
+```
+
+#### CloudRetro config
+
+First step is, to make our lives easier, to create a configMap the CloudRetro pods will be using. For that we need to download the original config file, and make a configMap Kubernetes resource from it. 
+
+```console
+wget https://raw.githubusercontent.com/giongto35/cloud-game/master/configs/config.yaml
+kubectl create configmap cloudretro-config --from-file=config.yaml --namespace=cloudretro
+```
+For the sepcific config details, please take a look at the original [file](https://github.com/giongto35/cloud-game/blob/master/configs/config.yaml).
+
+#### Cloudretro Coordinator deployment & service
+
+Now, we will need the container-image for CloudRetro. The simplest way is to download a very lightly modified version (for easier handling in k8s enviroment), but you can also build your own image [here](https://github.com/giongto35/cloud-game).
+
+After you got your image, we can jump right into deployments and pods. For this, I am using the image I`ve modified for this purpose (which is far from flawless, but it works).
+The following yaml file will make a deployment for the Coordinator, best to leave the scaling at 1 (TODO, worker sign in problems). By default, the Coordinator HTTP service will listen at port 8000, if you changed this in the config, make sure to change it here as well.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coordinator-deployment
+  namespace: cloudretro
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: coordinator
+  template:
+    metadata:
+      labels:
+        app: coordinator
+    spec:
+      containers:
+      - name: coordinator
+        image: valniae/snekyrepo:cloudimage4
+        volumeMounts:
+        - name: cloudretro-confmap-vol
+          mountPath: "/usr/local/share/cloud-game/configs/"
+          readOnly: true
+        command: ["coordinator"]
+        args: ["--v=5"]
+        ports:
+        - containerPort: 8000
+      volumes:
+      - name: cloudretro-confmap-vol
+        configMap:
+          name: cloudretro-config
+EOF
+```
+
+This should leave us with a working pod of coordinator:
+
+```console
+kubectl get pods -n cloudretro
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+coordinator-deployment-54hidoyoureadthis-qpnsx   1/1     Running   0          3m23s
+```
+
+Now that we have a running deployment of Coordinator(s), we will need a service for them. So in case a pod would die for whatever reason, the target address wouldn`t change for the users. In case you require, use a firewall-open TCP port for Nodeport, don`t forget to uncomment.
```

Comment:
Again, details are not interesting: make this into a yaml in the current dir and kubectl-apply that one. when I say "details are not interesting", I mean that in this examples/ dir the idea is to teach people to use STUNner and not necessarily to teach them to use cloudretro: the best practice is to give a high-level architecture at the beginning (see the request for an image at the top), and then whenever questions related to cloudretro come up (like "what is a cooddinator?", etc) just refer the reader back to the cloudretro docs